packages/brand-moment/src/stories/ExampleHeaders.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { assetUrl } from "@kaizen/hosted-assets"
+import styles from "./ExampleHeaders.scss"
+
+export const MinimalBasic = () => (
+  <div className={styles.header}>
+    <div className={styles.headerLeft}>
+      <img
+        src={assetUrl("brand/logo-horizontal-default.svg")}
+        alt="Culture Amp"
+        width={126}
+      />
```

Comment:
Should we default the logo to be a link to `/`? Or make it easy for it to people to set a link?

Reply:
🤔 these `ExampleHeaders` are just for Kaizen storybook purposes only, the real thing will actually be created on the consuming side, or at least passed in from the consuming side. I'm not sure if we'll always want this to be a link or not, but that can be worked out during implementation time.

Reply:
It’d be useful to show an example with a link at least I reckon.

Reply:
Done. FYI I am going to look at creating a helper/util component for adding focus indicators, which would be useful for links like this.